config/index.js

```diff
@@ -0,0 +1,8 @@
+const pkg = require('../package.json')
+
+const PORT = 5432
+const TEST_PORT = 5435
+
+const connectionString = `${PORT}/${pkg.name}-test`
```

Comment:
This won't quite work for our needs because `-test` is hard-coded in regardless of whether we want to use the test db or not. Using the test db should be contingent on some sort of env variable e.g. `process.env.NODE_ENV == 'test'` or `'testing'` or similar.